giotto/diagrams/tests/test_relevant_holes.py

```diff
@@ -0,0 +1,35 @@
+"""Testing for ReleventHoles"""
+
+import numpy as np
+import pytest
+from sklearn.exceptions import NotFittedError
+
+from giotto.diagrams import RelevantHoles
+
+X_rh = np.array([[[0, 1, 0], [0, 4, 0], [2, 3, 1], [0, 0, 2]],
+                 [[0, 3, 0], [0, 2, 0], [1, 2, 1], [0, 0, 2]]])
+
+
+def test_throw_exception_when_not_fitted():
+    rh = RelevantHoles()
+
+    with pytest.raises(NotFittedError):
+        rh.transform(X_rh)
+
+
+def test_output_for_relative_theshold():
```

Comment:
Typo

ENH: shorter name like `test_relative_threshold` (But we don't have any naming conventions)

Reply:
I believe it `test_output_for_relative_threshold` to be a better name in order to clarify what exactly is being checked (compare @lewtun comment/suggestion for `test_throw_exception_when_not_fitted` ). For example, in future, there could also be a shape test.